docs/getting_started.md

```diff
@@ -0,0 +1,262 @@
+# Getting Started
+
+This is a guide on how to get started with CAPV (Cluster API Provider vSphere).
+To learn more about cluster API in more depth, check out the the [cluster api docs page](https://cluster-api.sigs.k8s.io/).
+
+Table of Contents
+=================
+
+   * [Getting Started](#getting-started)
+   * [Table of Contents](#table-of-contents)
+      * [Bootstrapping a Management Cluster with clusterctl](#bootstrapping-a-management-cluster-with-clusterctl)
+         * [Install Requirements](#install-requirements)
+            * [Kind](#kind)
+            * [envsubst](#envsubst)
+            * [clusterctl](#clusterctl)
+         * [vSphere Requirements](#vsphere-requirements)
+            * [vCenter Credentials](#vcenter-credentials)
+            * [Networking Requirements](#networking-requirements)
+         * [Generating YAML for the Bootstrap Cluster](#generating-yaml-for-the-bootstrap-cluster)
+         * [Using clusterctl](#using-clusterctl)
+         * [Managing Workload Clusters using the Management Cluster](#managing-workload-clusters-using-the-management-cluster)
+
+## Bootstrapping a Management Cluster with clusterctl
+
+`clusterctl` is a command line tool used for [bootstrapping](https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/GLOSSARY.md#bootstrap) your [Mangement Cluster](https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/GLOSSARY.md#management-cluster).
+Your management cluster stores resources such as `clusters` and `machines` using the Kubernetes API. This is the cluster you use to provision and manange multiple clusters going forward.
+Before diving into the bootstrap process, it's worth noting that a Management Cluster is just another Kubernetes cluster with special addons and CRDs. You are not required to use `clusterctl` to
+provision your management cluster, however, this guide will be focused on using `clusterctl` to boostrap your management cluster using [Kind](https://github.com/kubernetes-sigs/kind).
+
+### Install Requirements
+
+#### Kind
+
+Assuming you have a Go installed on your machine, run the following to install Kind:
+
+```bash
+$ GO111MODULE="on" go get sigs.k8s.io/kind@v0.3.0
```

Comment:
Hi @andrewsykim,

Please change this to:

```shell
$ GO111MODULE="on" go get -mod readonly sigs.k8s.io/kind@v0.3.0
```

This ensures that installing Kind won't modify a project's Go module files.